spa/src/api/subscriptions/index.ts

```diff
@@ -1,5 +1,7 @@
+import { RestSyncReqBody } from "~/src/rest-interfaces";
 import { axiosRest } from "../axiosInstance";
 
 export default {
-	getSubscriptions: () => axiosRest.get("/rest/subscriptions")
+	getSubscriptions: () => axiosRest.get("/rest/subscriptions"),
+	syncSubscriptions: (data: RestSyncReqBody) => axiosRest.post(`/rest/app/cloud/sync`, data),
```

Comment:
It is similar to a comment I left for @krazziekay , I think we should follow the rest api semantic, so it would be `/rest/app/cloud/subscription/${subscriptionId}/sync`

Reply:
Using the installationId is wrong (in existing page) in my opinion, coz each entity IS a subscription, so we should natuarally deal with subscriptionId. Otherwise, when it comes to GHE, we will need to combine "installationId" and "gheAppId" together to avoid conflicts.